src/core.js

```diff
@@ -1,19 +1,22 @@
 define([
+	"./bundle/lookup",
 	"./common/create_error",
+	"./common/validate",
 	"./common/validate/presence",
 	"./common/validate/type",
 	"./common/validate/type/path",
 	"./common/validate/type/plain_object",
 	"./common/validate/type/string",
 	"./core/likely_subtags",
+	"./core/load",
 	"./core/remove_likely_subtags",
+	"./core/subtags",
 	"./item/get_resolved",
 	"./path/normalize",
 	"./resource/get",
 	"./util/always_array",
 	"./util/json/merge"
-], function( createError, validatePresence, validateType, validateTypePath,
-validateTypePlainObject, validateTypeString, coreLikelySubtags, coreRemoveLikelySubtags, itemGetResolved, pathNormalize, resourceGet, alwaysArray, jsonMerge ) {
+], function( bundleLookup, createError, validate, validatePresence, validateType, validateTypePath, validateTypePlainObject, validateTypeString, coreLikelySubtags, coreLoad, coreRemoveLikelySubtags, coreSubtags, itemGetResolved, pathNormalize, resourceGet, alwaysArray, jsonMerge ) {
```

Comment:
:laughing:, I tend to agree. But, throughout this code I've never broken lines for aesthetic reasons (although, I'm a fan of tw=80). I'll keep it this way for now unless this becomes epidemically bad. Thanks for pointing it out.